docs/course_materials/exercises/07_Exceptions/homework.md

```diff
@@ -0,0 +1,27 @@
+# Exceptions - HomeWork 
+
+### ex1_exceptions.py
+
+Create a new file ex1_exceptions.py Write a program that gets a number from the user, (use input function) 
```

Comment:
please break line 5 into 2 lines, also use the '-' for a new instruction line:
- Create a new file ex1_exceptions.py.
- Write a program that gets a number from the user, (use input function)